### PR TITLE
Add docs for rapt_ble integration

### PR DESCRIPTION
**Pull request by sairon**
## Proposed change

Add documentation for new rapt_ble integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/87872
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4119
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by thewolf-oz — DISMISSED** [review]
References all correct and process is sound

File: source/_integrations/rapt_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: RAPT Bluetooth
+description: Instructions on how to integrate RAPT Pill Hydrometers configured in Bluetooth mode into Home Assistant.
+ha_category:
+  - Sensor
+ha_bluetooth: true
+ha_release: '2023.03'
```
Comment (bdraco) [inline source/_integrations/rapt_ble.markdown]:
```suggestion
ha_release: 2023.5
```

File: source/_integrations/rapt_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates [RAPT Pill](https://www.kegland.com.au/rapt-pill-hydrometer-thermometer-wifi-bluetooth.html) hydrometers into Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+The RAPT Pill BLE integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional. RAPT Pill must be switched to the Bluetooth mode in Settings - Telemetry method. See the [user manual](https://www.kegland.com.au/media/pdf/KL20596%20-%20RAPT%20Pill%20Hydrometer%20and%20Thermometer%20Quick%20Start%20Guide.pdf) for detailed instructions.
```
Comment (bdraco) [inline source/_integrations/rapt_ble.markdown]: This link is stale

Comment (sairon) [reply source/_integrations/rapt_ble.markdown]: @bdraco Thanks, I've replaced it with updated link. KegLand recently switched to a new e-commerce solution, so let's hope this one lasts longer.

**Review by frenck — APPROVED** [review]
Thanks, @sairon 👍 

../Frenck

**Pull request merged**